tests/test_region_control.py

```diff
@@ -90,7 +90,7 @@ def close_windows(self):
             # HACK: make sure app is really closed
             time.sleep(0.5)
 
-    def wait_end(self, subprocess_pipe, timeout=30):
+    def wait_end(self, subprocess_pipe: bool, timeout: int = 30) -> bool:
```

Comment:
Let's revert this or just use `Any` for the pipe and `int` return arg

Reply:
Done